cypress/e2e/pages/safeapps.pages.js

```diff
@@ -0,0 +1,193 @@
+import * as constants from '../../support/constants'
+
+const searchappInput = 'input[id="search-by-name"]'
+const appUrlInput = 'input[name="appUrl"]'
+const closePreviewWindowBtn = 'button[aria-label*="Close"][aria-label*="preview"]'
+
+const addBtnStr = /add/i
```

Comment:
Nit: can we be a bit more specific that this is within the modal as there is also another add button in the list. (The same for the function which references this string.)